lib/opFns.js

```diff
@@ -581,7 +588,6 @@ module.exports = {
       })
 
       try {
-        checkCallMemCost(runState, options, localOpts)
```

Comment:
Yeah but it makes no sense as the `localOpts` are not changing at that point. Does the test expect gas deducted twice? The reason must be understood.